pstatus-notifications-workflow-ktor/src/main/kotlin/HealthCheck.kt

```diff
@@ -54,14 +55,16 @@ class HealthCheck {
 class HealthCheckService : KoinComponent {
     private val logger = KotlinLogging.logger {}
     private val databaseType: DatabaseType by inject()
+    private val temporalServiceTarget: String by inject() // Injected value
```

Comment:
Please don't inject a string, but rather a class type such as TemporalConfig, even if all it has is the target.  Who knows, maybe channel or something else will be eventually added to the config.  However, the main reason is that the inject is done by object type.  So, if for example, there is ever a second string injected (same object type), the inject will get confused which one to inject so it's better to have these be unique types.

Reply:
Got it. Will do